r/R/schema.R

```diff
@@ -50,6 +50,34 @@
 #' - `$metadata`: returns the key-value metadata as a named list.
 #'    Modify or replace by assigning in (`sch$metadata <- new_metadata`).
 #'    All list elements are coerced to string.
+#'
+#' @section Metadata:
```

Comment:
This section should either be called something like "R Metadata", or it should start by discussing the key-value metadata more generally.